docs/content/direct/release-testing.md

```diff
@@ -0,0 +1,29 @@
+# Testing a new KubeStellar Release
+
+The following testing process should be applied to every KubeStellar new release in order to validate the release. All the tests should be done while the KubeStellar code is still under code-freeze and new code shouldn't be merged into the main branch until all tests are passed and the release is declared as stable.  
+In case the release tests fail (even one of them), the release should be marked as instable and a fix should be worked on ASAP. The KubeStellar code-freeze should be lifted only after all tests are passed and a stable releases was released.  
+To reduce the exposure of instable releases the update of the KubeStellar site [kubestellar.io](https://docs.kubestellar.io) should be done only once all release tests passed sussceffuly. 
```

Comment:
BTW, the current release.md does not even mention updating the website's idea of which release is "stable". That is an oversight. To correct it, one would have to know how that updating is done. @clubanderson ?